src/tests/unit/components/app-button/AppButton.spec.jsx

```diff
@@ -0,0 +1,91 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import AppButton from '~/components/app-button/AppButton'
+import { vi } from 'vitest'
```

Comment:
First import libs and after components
```suggestion
import { vi } from 'vitest'
import { render, screen } from '@testing-library/react'
import userEvent from '@testing-library/user-event'
import AppButton from '~/components/app-button/AppButton'
```

Reply:
I'll take your suggestions into the considertaion for the future. Thank you!